lib/cn/kvset.c

```diff
@@ -1786,16 +1786,16 @@ kvset_stats_add(const struct kvset_stats *add, struct kvset_stats *result)
     result->kst_vgarb += add->kst_vgarb;
 }
 
-u64
-kvset_get_workid(struct kvset *ks)
+void *
+kvset_get_work(struct kvset *ks)
```

Comment:
nit: const

After multiple attempts, @tristan957 succeeded in making me pay attention to this :-)

Reply:
The world is a better place when things that can be const are!

Reply:
Grrrr, now I have to go fix the unit tests...

Reply:
Done.